app/views/acm/conference/about.haml

```diff
@@ -3,8 +3,53 @@
 %p
   Link-State is a student run conference put on every year by Case Western's ACM
   student group. The first Link-State conference was run October 27th 2012 and it
-  has been growing ever since. If you want to see what was happening at our previous
-  conferences, check out the archive below.
+  has been growing ever since. If you're curious as to where the name came from,
+  read its origin story below. If you want to see what was happening at our previous
+  conferences, check out the
+  %a{ href: '#archive' } archive
+  further down the page.
```

Comment:
I thought that's how you put links inside of p tags in haml. I'll try doing it inline.